mail/spring-cloud-starter-stream-source-mail/README.adoc

```diff
@@ -0,0 +1,27 @@
+//tag::ref-doc[]
+= Mail Source
```

Comment:
Looks like this this doc must be mentioned in the `spring-cloud-stream-app-starters-docs/../sources.adoc`